src/VersionHelper.php

```diff
@@ -6,13 +6,24 @@
 
 class VersionHelper
 {
-    public static function generateShortVersion(): string
-    {
-        return (new Version())->generate();
+    public static function generateShortVersion(
+        string $pathToVersionScript = Version::PATH_TO_VERSION_SCRIPT,
+        string $pathToCheckScript = Version::PATH_TO_CHECK_SCRIPT,
+    ): string {
+        return (new Version(
+            pathToVersionScript: $pathToVersionScript,
+            pathToCheckScript: $pathToCheckScript,
+        ))->generate();
```

Comment:
```suggestion
        return (new Version($pathToVersionScript, $pathToCheckScript))->generate();
```

variable names are descriptive enough here so we could use them as positional args instead of named ones